src/client/Microsoft.Identity.Client/Internal/JsonWebToken.cs

```diff
@@ -186,7 +186,7 @@ internal class JWTPayload
         [Preserve(AllMembers = true)]
         internal sealed class JWTHeaderWithCertificate : JWTHeader
         {
-            public JWTHeaderWithCertificate(ClientCredentialWrapper credential, bool sendCertificate)
+            public JWTHeaderWithCertificate(ClientCredentialWrapper credential, bool? sendCertificate = null)
```

Comment:
Can this be kept the same? We always pass `sendCertificate`.

Reply:
Yes.